<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringPattern</class>
 <widget class="QWidget" name="StringPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>98</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>151</width>
    <height>83</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Match type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pattern:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="inclusiveCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Include:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="patternLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="matchTypeCombo">
     <item>
      <property name="text">
       <string>Wildcard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regular Expression</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greedy Regular Expression</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unix Wildcard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W3C XML Syntax</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Case sensitive:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="casesensitiveCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
